src/frontend/ai-reviewer-admin/package.json

```diff
@@ -54,7 +55,8 @@
       "src/**/*.js",
       "!src/index.js",
       "!src/EnvConfig.js",
-      "!src/**/test-data.js"
+      "!src/**/test-data.js",
+      "!src/components/nav-drawer/NavDrawer.js"
```

Comment:
Why is NavDrawer.js excluded from the tests/coverage?

Reply:
Just fighting with the resize functionality of jest in order to test the nav, I think I've almost got it working though so I'll push an update shortly with the test